.github/workflows/build-rd-cpp.yml

```diff
@@ -0,0 +1,204 @@
+name: Build RD CPP
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  NINJA_VERSION: 1.9.0
+  BUILD_TYPE: Release
+  VERSION: 201.1
+  # Need to change these values to JetBrains account
+  # And set `RD_CPP_BINTRAY_KEY` for RD repo
+  BINTRAY_USER: alexander-pirogov
+  BINTRAY_REPO: RD-CPP
+
+jobs:
+  build:
+    env:
+      working-dir: ${{ github.workspace}}${{ matrix.config.SEP }}rd-cpp
+    name: ${{ matrix.config.name }}
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - {
+            name: "RD CPP Windows Latest MSVC", artifact: "RD-CPP-Windows-MSVC-2019.zip",
+            os: windows-latest,
+            cc: "cl", cxx: "cl",
```

Comment:
Is newline required after cc?

Reply:
yaml works just fine with this formatting.
Same on line 29, where `name` and `artifact` on the same line

Reply:
ok